File: Java/Base/Sort.java
import java.util.*;
public class Sort{
    public static void main(String[] args){
        int arr[] = {2,5,4,2,3,4,2};
        for(int i = arr.length -1 ; i > 0; i--){
            for(int j = 0; j < i; j++){
                if(arr[j] > arr[j+1]){
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j + 1] = temp;
                }
            }
        }
            System.out.println(Arrays.toString(arr));
    }
}

File: Python/Base/Sort.py
arr:list[int] = [36,65,3,4,265,31,2,986,32]

for i in range(len(arr)-1,-1,-1):
    for j in range(0,i):
        if arr[j] > arr[j + 1]:
            temp : int = arr[j]
            arr[j] = arr[j + 1]
            arr[j + 1] = temp

